minio/api.py

```diff
@@ -507,9 +507,10 @@ def listen_bucket_notification(self, bucket_name, prefix='', suffix='',
                                       query=query, preload_content=False)
             try:
                 for line in response.stream():
-                    event = json.loads(line)
-                    if event['Records'] is not None:
-                        yield event
+                    if len(line.strip()) > 0:
```

Comment:
It is just a preference, but IMHO, `if not line.strip():` is better than checking the length of a string.

Reply:
yes @ebozduman this is much simpler. Changing it

Reply:
@ebozduman  Done